platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationFileInstaller.java

```diff
@@ -236,6 +236,14 @@ private static ZipFileApplicationDetails getAppDetailsFromFeature(ZipFile zipFil
         }
 
         DocumentBuilderFactory dbFactory = DocumentBuilderFactory.newInstance();
+
+        try {
+            dbFactory.setFeature("http://apache.org/xml/features/nonvalidating/load-dtd-grammar", false);
```

Comment:
Specifically, I think we could use https://xerces.apache.org/xerces2-j/javadocs/xerces2/org/apache/xerces/impl/Constants.html since we provide that package in our custom.properties.